content/topics/redux-thunks/_index.md

```diff
@@ -6,7 +6,17 @@ prerequisites:
   - topics/redux-intro
   soft: []
 title: Redux Thunks
+ready: True
 ---
 
+Redux Thunk is a really confusing thing when you first hear about it.especailly that word “thunk.” So let’s clear that up first.
+
+Redux Thunk middleware allows you to write action creators that return a function instead of an action. The thunk can be used to delay the dispatch of an action, or to dispatch only if a certain condition is met,so instead of performing some work now, you produce a function body or unevaluated expression (the “thunk”) which can optionally be used to perform the work later.
+
+## Why are Redux-Thunks useful?
+
+Without middleware, Redux store only supports synchronous data flow.Thunks extend the store's abilities, and lets you write async logic that interacts with the store.
```

Comment:
Punctuation is a bit strange. Aways remember to have a space after a full stop.

Here is an example.This iexample is bad
Here is another example. This one is good because there is a space